net/tayga/src/etc/inc/plugins.inc.d/tayga.inc

```diff
@@ -70,9 +70,9 @@ function tayga_interfaces()
         return $interfaces;
     }
     $oic = array('enable' => true);
-    $oic['if'] = 'nat64';
+    $oic['if'] = 'tayga';
```

Comment:
actually, pretty funny this uses "nat64" as it aligns with name+number scheme

Reply:
Could this be a problem? This is just the interface name we configure here:
https://github.com/opnsense/plugins/blob/926d426442d2e3401d0e296ee54d512de3137dce/net/tayga/src/opnsense/service/templates/OPNsense/Tayga/tayga.conf#L3
`nat64` is what's used as an example in the official Tayga readme, but we could change that to something else if required.

Reply:
no, it's fine 👍